Clarified means of "store records" setting in Umbraco Forms when it comes to file uploads. (#6255)
umbraco/UmbracoDocs: 10/umbraco-forms/editor/creating-a-form/form-settings.md

### Store Records

By default, all submitted records are saved in the database. This option allows you to view and export the saved records from the queries overview. If you do not want to store data (due to policies in your organization), you can uncheck the box.

> Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission are retained. If you do not want these stored, any method used to process or copy them to a different location should also remove the file.

Reviewer: ```suggestion
Disabling this option will prevent database records from being stored, but any file uploads made as part of the form submission will still be retained. If you do not want the files to be stored, ensure that any process or method used to process, move, or copy them to a different location also removes the file.
```
